test/integration/puppetlabs/services/jruby/tasks_test.clj

```diff
@@ -58,8 +74,8 @@
                   (str module-name "::" name))})
        tasks))
 
-(deftest ^:integration all-tasks-test
-  (testing "requesting all tasks"
+(deftest ^:integration task-data-test
```

Comment:
The diff is confusing here. The all-tasks-test is below this one, I didn't replace it.